core/sets.gms

```diff
@@ -2501,21 +2493,20 @@ VRE2teVRElinked(all_te,all_te)    "mapping between the technologies requiring st
 VRE2teGrid(all_te,teGrid)              "mapping to know which technology needs which grid technology (length/siting)"
 /
         spv.gridspv
-        wind.gridwind
-$IFTHEN.WindOff %cm_wind_offshore% == "1"
+***        wind.gridwind
+        windon.gridwindon
         windoff.gridwindoff
-$ENDIF.WindOff
         csp.gridcsp
 /
 
 te2teLoclinked(teLoc,teLoc2)   "mapping between the technologies requiring grids which use similarly sited resources (so the shareseel counts 1/4 towards the other shareseel)"
```

Comment:
could you remove this mapping? grid is by now not differentiated between tech anymore.